dev/com.ibm.ws.ejbcontainer.security.jacc_fat.2.1/fat/src/com/ibm/ws/ejbcontainer/security/jacc_fat/FATSuite.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2012, 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.ejbcontainer.security.jacc_fat;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import componenttest.rules.repeater.EmptyAction;
+import componenttest.rules.repeater.JakartaEE10Action;
+import componenttest.rules.repeater.RepeatTests;
+
+@RunWith(Suite.class)
+@SuiteClasses({
+                EJBJakarta10Test.class
+})
+
+public class FATSuite {
+    private static final Set<String> EE78_FEATURES;
+    private static final String[] EE78_FEATURES_ARRAY = {
+                                                          "appSecurity-1.0",
+                                                          "usr:jaccTestProvider-1.0"
+    };
+
+    private static final Set<String> EE9_FEATURES;
+    private static final String[] EE9_FEATURES_ARRAY = {
+                                                         "usr:jaccTestProvider-2.0"
+    };
+
+    private static final Set<String> EE10_FEATURES;
+    private static final String[] EE10_FEATURES_ARRAY = {
+                                                          "usr:jaccTestProvider-2.1"
+
+    };
+
+    static {
+        EE78_FEATURES = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(EE78_FEATURES_ARRAY)));
+        EE9_FEATURES = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(EE9_FEATURES_ARRAY)));
+        EE10_FEATURES = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(EE10_FEATURES_ARRAY)));
+    }
+
+    /*
+     * Run EE10 tests in LITE mode and run all tests in FULL mode.
+     */
+    @ClassRule
+    public static RepeatTests repeat = RepeatTests.with(new EmptyAction().fullFATOnly()).andWith(new JakartaEE10Action().removeFeatures(EE78_FEATURES).removeFeatures(EE9_FEATURES).addFeatures(EE10_FEATURES));
```

Comment:
```suggestion
    public static RepeatTests repeat = RepeatTests.with(new JakartaEE10Action());
```
It would seem you only need this.  If the server.xml files are already EE10 ready, you don't need the extra stuff.